schema/xsd/portal.xsd

```diff
@@ -275,6 +275,14 @@ Currently known values:
         The signer has signed the job.
     * BLOCKED
         The signer entered the wrong security code too many times. Only applicable for signers addressed by e-mail address or mobile number.
+    * SIGNERS_NAME_NOT_AVAILABLE
```

Comment:
Når denne statusen kommer så kan de vel prøve på nytt å hente status og så kan det ha endret seg, om jeg har forstått rett? Dvs om henting av navn feilet midlertidig. Kanskje vi burde ha med noe om det i tillegg her da?

Reply:
For asynkrone oppdrag er det utelukkende permanente feil som fører til denne statusen. Hvis det er en forbigående feil blir ikke oppdraget avbrutt og undertegner kan prøve igjen. Avsender blir ikke informert om dette.

Reply:
Right, da skjønner jeg!